drydock/plugin.py

```diff
@@ -2,7 +2,7 @@
 import functools
 import os
 import click
-import pkg_resources
+import importlib_resources
```

Comment:
Why `importlib_resources` over `importlib.resources`?

Reply:
This is what is used by all tutor [plugins](https://github.com/overhangio/cookiecutter-tutor-plugin/blob/master/%7B%7B%20cookiecutter.package_name%20%7D%7D/%7B%7B%20cookiecutter.module_name%20%7D%7D/plugin.py#L7).

Reply:
Seems like the `importlib_resources` package was added to tutor dependencies in [`v17.0.2`](https://github.com/overhangio/tutor/commit/d99b2feeb30fa772dca48345c8992b4eafdeac21) does this break on previous tutor versions?

Reply:
Yes, it breaks:

![image](https://github.com/eduNEXT/drydock/assets/33465240/12051ce1-a6d4-41c6-8939-d30a61c7cdd1)

Reply:
However, this is not a problem as this change is specific the drydock latest version

Reply:
But we should still update the setup.py (https://github.com/eduNEXT/drydock/blob/main/setup.py#L42).